phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1386,7 +1430,7 @@ public MutationState createFunction(CreateFunctionStatement stmt) throws SQLExce
                 FunctionArgument arg = args.get(i);
                 addFunctionArgMutation(function.getFunctionName(), arg, argUpsert, i);
             }
-            functionData.addAll(connection.getMutationState().toMutations().next().getSecond());
+            functionData.addAll(connection.getMutationState().toMutations(null).next().getSecond());
```

Comment:
This toMutations(null) looks a little funky. Maybe overload it so the null isn't required?